recipes/fastscapelib-f2py/build.sh

```diff
@@ -1,7 +1,6 @@
 if [ $(uname) == Darwin ]; then
     # directly setting MACOSX_DEPLOYMENT_TARGET is not supported by scikit-build
     $PYTHON -m pip install . \
-            --global-option="build_ext" \
```

Comment:
Sorry, the equal sign should not be used. 

```
--global-option="--plat-name macosx-$MACOSX_DEPLOYMENT_TARGET-$OSX_ARCH"
```

(For info, see https://scikit-build.readthedocs.io/en/latest/generators.html#customizing-deployment-target-and-architecture)

Reply:
Unfortunately the option is not recognized either without the equal sign.

Reply:
I see .. this means that global option would need to be specified like:

```
--global-option="--plat-name" --global-option="macosx-$MACOSX_DEPLOYMENT_TARGET-$OSX_ARCH"
```

Reply:
After some more testing, the `--plat-name` option must be associated with either `bdist_wheel` or `build_ext` command, I will clarify the documentation.

What you add originally was correct  :unamused:

Reply:
OK, I'll revert it to the original addition. Thanks for trying anyway, it's always good to know ;-).